tutorials/viewports/multiple_resolutions.rst

```diff
@@ -3,21 +3,24 @@
 Multiple resolutions
 ====================
 
-Base resolution
----------------
+Base size
+---------
 
-A base screen resolution for the project can be specified in the project
-settings under ``display``, ``window``
+A base size for the project can be specified in the project settings
+under "Display", "Window".
```

Comment:
There's not a hard "standard", but most of the rest of the docs use **Project Settings** for the settings window and "Display/Window" to refer to the subsections to distinguish where to go.